### PR TITLE
Added a new Enter Handler and multiple input key press event handlers

### PR DESCRIPTION
**Pull request by Nopraz**
- Now if the user press enter on the 4 inputs the same event as the
buttons next to theses inputs triggers
- Renamed the duplicated ids on the html for cleaning purpose

### CODE REVIEW

File: todos.js [hunk 1]
```diff
@@ -180,6 +179,48 @@ var handler = {
...
+	enterHandler(e.key, handler.changeTodos, []);
+};
+
+//Enter event handler for change inputs
+let todoChangesInputs = document.querySelectorAll('.todoChangeInputs');
+let todoChangesInputsLength = todoChangesInputs.length;
+
+for(let i = 0; i < todoChangesInputsLength; i++){
+	todoChangesInputs[i].addEventListener("keypress", todoChangePositionHandler);
+}
```
Comment (blacksag) [inline todos.js]: The event listener is added for both the input fields of "Change" functionality, which might call the corresponding handler on pressing enter on either one of input fields while the other being empty and thus causing problem in handler function running on null values.

**Review by blacksag — COMMENTED** [review]
Event handlers for change functionality are not as required.

File: todos.js (hunk 1 above)
Comment (Nopraz) [reply todos.js]: Yeah you are right, i will change that for the last change input only

Comment (Nopraz) [reply todos.js]: Changed this on commit f83c45a

**Review by blacksag — APPROVED** [review]

**Pull request merged**